_documentation/user/flags/ricochet.md

```diff
@@ -5,4 +5,12 @@ type: Good
 summary: Shots bounce off walls. Don't shoot yourself!
 ---
 
-...
+The Ricochet (R) flag allows the player to fire shots that can bounce off of the objects they hit. The direction of the bounce is determined by the angle under which the platform is hit. An exception are objects with the attributes `shootthrough` or `passable`, where the result is the bullet passing through the object.
```

Comment:
> The direction of the bounce is determined by the angle under which the platform is hit.

This is awkward. What platform and why am I hitting under it? I think you may be trying to allude to the [Law of Reflection](https://www.physicsclassroom.com/Class/refln/u13l1c.cfm) but it's not clear.

Reply:
@jwmelto valid point; will elaborate. Any concrete suggestions beforehand?

Reply:
well, besides being somewhat obvious basic physics, "the angle of reflection is equal to the angle of incidence" comes to mind.

Reply:
I was asking for a concrete example here to avoid potential further displeasure in the wording, as we are basically thinking of the same thing just the wording was off.

Will use your suggestion.